### PR TITLE
Trello bug fix & updates

### PR DESCRIPTION
**Pull request by michelle0927**
PR to address issue: https://github.com/PipedreamHQ/pipedream/issues/1868

### CODE REVIEW

**Comment by dylburger** [comment]
- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling soures, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

**Review by js07 — APPROVED** [review]
The bug fix and updates to ESM look good and all of the sources work for me.

File: components/trello/sources/card-due-date-reminder/card-due-date-reminder.mjs [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
   dedupe: "unique",
   props: {
     ...common.props,
     board: {
-      propDefinition: [common.props.trello, "board"],
+      propDefinition: [
+        common.props.trello,
+        "board",
+      ],
       default: "me",
```
[the remark this answers is not in the document]
Comment (michelle0927) [reply components/trello/sources/card-due-date-reminder/card-due-date-reminder.mjs]: Yep, you're right. Updated this to remove the default.

**Review by js07 — APPROVED** [review]
lgtm

**Comment by dannyroosevelt** [comment]
@michelle0927 I selected `actions`, `list`, and `card` when running `pd dev` in my terminal, but when I then opened the source in my browser, none of those were selected and I have this error in the logs:
image

**Comment by michelle0927** [comment]
@dannyroosevelt I wasn't able to reproduce that exactly, but I did see that error occurring when no Event Types are selected. Made a small update to fix it that checks that the incoming event body contains `action`.

**Comment by dannyroosevelt** [comment]
> @dannyroosevelt I wasn't able to reproduce that exactly, but I did see that error occurring when no Event Types are selected. Made a small update to fix it that checks that the incoming event body contains `action`.

Hah I think the specific issue I cited was actually user error w/ the CLI 😬🤦🏻‍♂️

This lgtm! Thanks for fixing, @michelle0927 feel free to merge!

**Pull request merged**